content/javascript/concepts/promise/terms/then/then.md

```diff
@@ -44,3 +44,17 @@ myPromise.then(
   }
 );
 ```
+
+## Codebyte Example
+
+The following example shows a block scoped variable declared using the let keyword.
```

Comment:
The introduction of this codebyte should be talking about what `.then()` is doing rather than the scope of the variables.

Reply:
Thanks for your feedback. I will make the necessary change and commit again. Do I need to push the changes to the existing branch or create a new one?